### ISSUE TITLE
create deployments

### ISSUE THREAD

**Issue by MarcAntoine-Arnaud** [mikrosimage/loudness_validator#14]
1. create rule for homebrew
2. create debian packaging

**Comment by cchampet** [comment]
First step on Mac OS X [here](https://github.com/cchampet/homebrew-video/commit/d11f6c6b35ff37584d3dfba494e0972ceb9650b6). 
Maybe need to rename ```homebrew-video``` to ```homebrew-media```, or create a new one as```homebrew-audio```? This library is ok here for you @cbenhagen?

**Comment by cbenhagen** [comment]
I am ok with having this formula in there. and maybe a rename to homebrew-media but maybe you could just add it to homebrew itself. I feel like we should try to do this for all our formulae.

**Comment by cchampet** [comment]
Agreed. I will push this formula to homebrew-video, and in a second time we will propose the formulas to homebrew itself (or propose to add a repository ```Homebrew/homebrew-video``` like they did for science, games, python...).

**Comment by cbenhagen** [comment]
If possible i would prefer not have 'Homebrew/homebrew-video' but rather have it in homebrew itself. That way its easier for the user and we can profit from the homebrew buildbot and more review of our formulae. I only started my own tap because my formulae were not ready to get upstream. But i think it would make sense to move all of the formulae upstream when ready.

**Comment by MarcAntoine-Arnaud** [comment]
I agreed with you Ben, it will be very fine to push it directly to homebrew.
Have you already do that before ?
Do you now what we need to give exactlly ?

Cheers,
Marc-Antoine


Le mar. 15 sept. 2015 à 14:33, Ben Hagen <notifications@github.com> a
écrit :

**Comment by cbenhagen** [comment]
Yes I have. Its basicalliy the same as with my repo. You just submit a pull request like `"foobar 7.3 (new formula)"`. They will be more strict about things like style and tests and so then me. But this will only benefit our formulae. So another reason to put it in main homebrew.

Be sure to `brew audit foobar` before the pull request.

**Comment by cbenhagen** [comment]
You can find all the information needed in: https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Formula-Cookbook.md
